index.html

```diff
@@ -185,5 +185,25 @@ <h2 class="third-text"><a>let's connect</a></h2>
           </li>
         </ul>
     </section>
+    <section class="contact-form">
+        <ul>
+            <li><img class="ellipse-12-2" src="elements/ellipse-12-2.png" alt="icon-7"></li>
+            <li><img class="ellipse-5-w" src="elements/ellipse-5-white.png" alt="icon-8"></li>
+            <li><img class="ellipse-3-w" src="elements/ellipse-3-white.png" alt="icon-9"></li>
+            <li><img class="rect-4-d" src="elements/rectangle-4-dark.png" alt="icon-10"></li>
+            <li><img class="rect-7-2" src="elements/rectangle-7-2.png" alt="icon-11"></li>
+        </ul>
+        <h2 class="contact-title">Contact me</h2>
+       <h3 class="contact-subtitle">If you have an application you are interested in developing, 
```

Comment:
- [ ] **OPTIONAL** Please notice that this paragraph is supposed to be a `<p>` tag, and not a `<hx>` because it's not a heading. It's so important to follow the [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) concepts.